.github/workflows/actions/docker-images-save/action.yml

```diff
@@ -0,0 +1,21 @@
+name: Action > Docker Images Build
+inputs:
+  image-build-tag:
+    required: true
+runs:
+  using: 'composite'
+  steps:
+    - uses: ./.github/workflows/actions/cached-dependencies
+    - uses: ./.github/workflows/actions/load-conf-env
+    - name: Build Docker Save
+      run: |
+        set -eo pipefail
+        source $GITHUB_WORKFLOW_FOLDER/actions/docker-images-save/saveImages.sh "${{ inputs.image-build-tag }}" "./.github/workflows/actions/docker-images-save/docker-images"
+      shell: bash
+    - uses: actions/upload-artifact@v4
+      with:
+        name: docker-images-${{ inputs.image-build-tag }}
+        path: |
+          ./.github/workflows/actions/docker-images-save/docker-images/
+        retention-days: 2
```

Comment:
Do we really need to store these artifacts? Why not rather push to dockerhub with a commit hash as the tag? Then they are always available.

Reply:
It's mainly there for debugging purpose (hence the rentention only 2 days). Regarding pushing to docker hub, I think the plan was only to push images for main/releases, least dockerhub becomes too convoluted (and could also end up includes bugy images). @w-hayes thoughts?

Reply:
I just think we are going to pay for storing these. I think pushing them to a tag or doing not saving at all makes the most sense.

Reply:
Okay, will make it manual workflow (in case someone wants to use it to debug), and remove it from the event driven workflows. If this fine?

Reply:
Cool, that sounds fine.